Feature: Update README (#238)
itk-dev/economics: README.md

APP_INVOICE_DEFAULT_DESCRIPTION=<VALUE>

###> symfony/mailer ###
MAILER_DSN=smtp://mail:1025
###< symfony/mailer ###
```

### Data

You have two options to get some data into the database.

You can either:

- [Load in fixtures](#fixtures)
- [Synchronize data](#synchronize-data) from an actual project tracker.

#### Fixtures

To load fixtures

```sh
task fixtures:load
```

#### Synchronize Data

##### Data provider

To synchronize data, we need to setup a data provider.

Create data-provider

```sh
task data-provider:create
```

- Enter name
- Enter the base url of the project tracker(e.g. <https://leantime.whatever.com>)
- Enter secret
- Use up-down arrows to select an implementation class
  - A class is implemented for each supported project tracker

##### Synchronize projects

Before issues and worklogs can be synchronized, projects need to be synchronized and "included".

Queue project and account sync jobs and consume the jobs

```sh
task queue:projects
task queue:accounts
task queue:consume-all
```

- Navigate to /admin/project/ and filter by "Not included"
- Include the projects you want to synchronize data for.

##### Synchronize issues/worklogs

> After including the projects you want to synchronize data for, you can synchronize issues and worklogs.

Reviewer: ```suggestion
After including the projects, you can synchronize issues and worklogs for these projects.
```
